.github/pre-commit-config.yaml

```diff
@@ -0,0 +1,13 @@
+repos:

```

Comment:
Shouldn't precommit be included in the yaml file?

Reply:
Do you mean other precommit than black, flake8 or mypy? These ones are included